src/store/reducers/booksReducer.ts

```diff
@@ -1,11 +1,12 @@
-import { BookActionTypes } from '../types/books';
+import { BookActionTypes } from "../types/books";
 
 const defaultState = {
   books: [],
   currentPage: 1,
   totalCount: 0,
-  query: "all",
+  query: "", //! не менять это значение!! нужно для корректной работы постраничнойпагинации
```

Comment:
I know it's now local comment, but try to leave such indicate comments in English